<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-28T19:13:24 IST" name="Suite" finished-at="2022-02-28T19:13:33 IST" duration-ms="8980">
    <groups>
    </groups>
    <test started-at="2022-02-28T19:13:24 IST" name="Text Verificastion on homepage" finished-at="2022-02-28T19:13:29 IST" duration-ms="4932">
      <class name="Academy.homePage">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.homePage@45815ffc]" started-at="2022-02-28T19:13:24 IST" name="initialize" finished-at="2022-02-28T19:13:29 IST" duration-ms="4459" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="testingLandingpage()[pri:0, instance:Academy.homePage@45815ffc]" started-at="2022-02-28T19:13:29 IST" name="testingLandingpage" finished-at="2022-02-28T19:13:29 IST" duration-ms="66" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testingLandingpage -->
        <test-method signature="testvideoBanner()[pri:0, instance:Academy.homePage@45815ffc]" started-at="2022-02-28T19:13:29 IST" name="testvideoBanner" finished-at="2022-02-28T19:13:29 IST" duration-ms="53" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testvideoBanner -->
        <test-method is-config="true" signature="closeinstance()[pri:0, instance:Academy.homePage@45815ffc]" started-at="2022-02-28T19:13:29 IST" name="closeinstance" finished-at="2022-02-28T19:13:29 IST" duration-ms="344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeinstance -->
      </class> <!-- Academy.homePage -->
    </test> <!-- Text Verificastion on homepage -->
    <test started-at="2022-02-28T19:13:24 IST" name="Validating inputs with 2 entries" finished-at="2022-02-28T19:13:32 IST" duration-ms="7841">
      <class name="Academy.homePage2">
        <test-method is-config="true" signature="close_All_Browser_Instances()[pri:0, instance:Academy.homePage2@293a5f75]" started-at="2022-02-28T19:13:33 IST" name="close_All_Browser_Instances" finished-at="2022-02-28T19:13:33 IST" duration-ms="97" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close_All_Browser_Instances -->
      </class> <!-- Academy.homePage2 -->
      <class name="Academy.BaseTest">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.BaseTest@681a8b4e]" started-at="2022-02-28T19:13:24 IST" name="initialize" finished-at="2022-02-28T19:13:28 IST" duration-ms="4154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="testingLogindetails(java.lang.String,java.lang.String)[pri:0, instance:Academy.BaseTest@681a8b4e]" started-at="2022-02-28T19:13:28 IST" name="testingLogindetails" data-provider="getData" finished-at="2022-02-28T19:13:30 IST" duration-ms="1620" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[test]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Testpassword]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Academy.BaseTest.testingLogindetails(BaseTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testingLogindetails -->
        <test-method signature="testingLogindetails(java.lang.String,java.lang.String)[pri:0, instance:Academy.BaseTest@681a8b4e]" started-at="2022-02-28T19:13:30 IST" name="testingLogindetails" data-provider="getData" finished-at="2022-02-28T19:13:32 IST" duration-ms="1532" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[tests]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Testpassword2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Academy.BaseTest.testingLogindetails(BaseTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testingLogindetails -->
        <test-method is-config="true" signature="closeinstance()[pri:0, instance:Academy.BaseTest@681a8b4e]" started-at="2022-02-28T19:13:32 IST" name="closeinstance" finished-at="2022-02-28T19:13:32 IST" duration-ms="71" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeinstance -->
      </class> <!-- Academy.BaseTest -->
    </test> <!-- Validating inputs with 2 entries -->
    <test started-at="2022-02-28T19:13:24 IST" name="Presence of navigation bar" finished-at="2022-02-28T19:13:33 IST" duration-ms="8980">
      <class name="Academy.homePage2">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Academy.homePage2@293a5f75]" started-at="2022-02-28T19:13:24 IST" name="initialize" finished-at="2022-02-28T19:13:29 IST" duration-ms="4374" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="testingNevigationBar()[pri:0, instance:Academy.homePage2@293a5f75]" started-at="2022-02-28T19:13:29 IST" name="testingNevigationBar" finished-at="2022-02-28T19:13:29 IST" duration-ms="151" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testingNevigationBar -->
        <test-method is-config="true" signature="closeinstance()[pri:0, instance:Academy.homePage2@293a5f75]" started-at="2022-02-28T19:13:29 IST" name="closeinstance" finished-at="2022-02-28T19:13:33 IST" duration-ms="4443" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeinstance -->
      </class> <!-- Academy.homePage2 -->
    </test> <!-- Presence of navigation bar -->
  </suite> <!-- Suite -->
</testng-results>
